### PR TITLE
Nc news task 6

### PR DESCRIPTION
**Pull request by Northerner988**

### CODE REVIEW

**Review by Keb84 — CHANGES_REQUESTED** [review]
I'm currently getting errors in the console from your SingleArticle when i slow the network down to see your loading state. Please fix these errors and resubmit a PR 👍

File: src/components/CommentList.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return (
+    <ul className="comment-container">
+      {isLoading ? (
+        <p>Fetching comments...</p>
+      ) : (
+        currentComments.map((comment) => (
+          <CommentCard key={comment.comment_id} comment={comment.body} />
+        ))
+      )}
+    </ul>
```
Comment (MKD01) [inline src/components/CommentList.jsx]: One thing to think about here is what if an article has no comments? What is displayed to the user in that situation?

**Review by MKD01 — APPROVED** [review]
Great work so far, I have left a comment for you as well and once you have made the change for that im happy for you to merge this in 👍

File: src/components/CommentList.jsx (hunk 1 above)
Comment (Northerner988) [reply src/components/CommentList.jsx]: Okay thank you! I will add an if statement to account for this situation.

**Pull request merged**